docs/reference/ingest/apis/enrich/put-enrich-policy.asciidoc

```diff
@@ -0,0 +1,143 @@
+[[put-enrich-policy-api]]
+=== Put enrich policy API
+++++
+<titleabbrev>Put enrich policy</titleabbrev>
+++++
+
+Creates an enrich policy.
+
+////
+[source,js]
+----
+PUT /users
+----
+// CONSOLE
+////
+
+[source,js]
+----
+PUT /_enrich/policy/my-policy
+{
+    "match": {
+        "indices": "users",
+        "match_field": "email",
+        "enrich_fields": ["first_name", "last_name", "city", "zip", "state"]
+    }
+}
+----
+// CONSOLE
+// TEST[continued]
+
+
+[[put-enrich-policy-api-request]]
+==== {api-request-title}
+
+`PUT /_enrich/policy/<enrich-policy>`
+
+
+[[put-enrich-policy-api-prereqs]]
+==== {api-prereq-title}
+
+// tag::enrich-policy-api-prereqs[]
+If you use {es} {security-features}, you must have:
+
+* `read` index privileges for any indices used
+* The `enrich_user` {stack-ov}/built-in-roles.html[built-in role]
+// end::enrich-policy-api-prereqs[]
+
+
+[[put-enrich-policy-api-desc]]
+==== {api-description-title}
+
+Use the put enrich policy API
+to create a new enrich policy.
+
+// tag::enrich-policy-def
+An *enrich policy* is a set of rules the enrich processor uses
+to append the appropriate data to incoming documents.
+An enrich policy contains:
+
+* The *policy type*,
+  which determines how the processor enriches incoming documents
+* A list of source indices
+* The *match field*, a field used to match incoming documents
+* *Enrich fields*, fields appended to incoming documents
+  from matching documents
+// end::enrich-policy-def
+
+
+===== Update an enrich policy
+
+// tag::update-enrich-policy
+You cannot update an existing enrich policy.
+Instead, you can:
+
+.   Create a new enrich policy
```

Comment:
Maybe change `Create a new enrich policy` into `Create and execute a new enrich policy`?
The new policy cannot be configured in a pipeline if it hasn't been executed before.

Reply:
Fixed with 928f62e329aff270e78a986f2ad0a733cef1f6c4.